tests/fiaas_skipper/test_update.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+import pytest
+from mock import mock, patch, MagicMock
+
+from fiaas_skipper import AutoUpdater, update
```

Comment:
`AutoUpdater` is defined in `fiaas_skipper.update`, and is just by coincidence imported in `fiaas_skipper`. I would import it from where it is defined to avoid the import time dependency on `fiaas_skipper`. The way it is done right now would cause this to `ImportError` if `AutoUpdater` is no longer used in `fiaas_skipper` which could be confusing